### PR TITLE
GM: Fix wheel direction signal

### PR DESCRIPTION
**Pull request by nworb-cire**

### CODE REVIEW

**Comment by sshane** [comment]
For https://github.com/commaai/openpilot/pull/31240

**Comment by sshane** [comment]
Can you test with our segments in routes.py which cars it's unknown or not supported? You can add an assert in the openpilot PR and it will print which fail

**Comment by nworb-cire** [comment]
All GM vehicles except the Trailblazer have a value of 0 at some point (not sure if that one just isn't tested). The Equinox seems to be the only one with a value of 3, although I can say off the top of my head that several non-ACC cars (Bolts, to memory) from about the same time period also have values of 3.

**Comment by sshane** [comment]
Looks like unknown is just "none":

image

File: generator/gm/gm_global_a_powertrain.dbc [hunk 1]
```diff
@@ -331,8 +331,8 @@ VAL_ 452 CruiseState 4 "Standstill" 3 "Faulted" 1 "Active" 0 "Off" ;
 VAL_ 309 PRNDL 3 "R" 2 "D" 1 "N" 0 "P" ;
 VAL_ 309 ESPButton 1 "Active" 0 "Inactive" ;
 VAL_ 384 LKASteeringCmdActive 1 "Active" 0 "Inactive" ;
-VAL_ 842 RRWheelDir 0 "Unknown" 1 "Forward" 2 "Reverse" 3 "Unsupported" 4 "Fault";
-VAL_ 842 RLWheelDir 0 "Unknown" 1 "Forward" 2 "Reverse" 3 "Unsupported" 4 "Fault";
+VAL_ 842 RRWheelDir 0 "Stationary" 1 "Forward" 2 "Reverse" 3 "Unsupported" 4 "Fault";
+VAL_ 842 RLWheelDir 0 "Stationary" 1 "Forward" 2 "Reverse" 3 "Unsupported" 4 "Fault";
```
Comment (sshane) [inline generator/gm/gm_global_a_powertrain.dbc:335]: can keep it unknown

Comment (sshane) [reply generator/gm/gm_global_a_powertrain.dbc:335]: nvm this is prob fine

File: generator/gm/gm_global_a_powertrain.dbc [hunk 2]
```diff
@@ -228,12 +228,10 @@ BO_ 840 EBCMWheelSpdFront: 5 K17_EBCM
...
- SG_ RLWheelSpd : 7|16@0+ (0.0311,0) [0|255] "km/h"  NEO
- SG_ RRWheelSpd : 23|16@0+ (0.0311,0) [0|255] "km/h"  NEO
- SG_ MovingForward : 32|1@0+ (1,0) [0|1] "" XXX
- SG_ MovingBackward : 33|1@0+ (1,0) [0|1] "" XXX
- SG_ MovingForward2 : 35|1@1+ (1,0) [0|1] "" XXX
- SG_ MovingBackward2 : 36|1@0+ (1,0) [0|1] "" XXX
+ SG_ RLWheelSpd : 7|16@0+ (0.0311,0) [0|255] "km/h" NEO
+ SG_ RRWheelSpd : 23|16@0+ (0.0311,0) [0|255] "km/h" NEO
+ SG_ RLWheelDir : 37|3@0+ (1,0) [0|7] "" NEO
+ SG_ RRWheelDir : 34|3@0+ (1,0) [0|7] "" NEO
```
Comment (sshane) [inline generator/gm/gm_global_a_powertrain.dbc:234]:
```suggestion
 SG_ RRWheelDir : 34|3@0+ (1,0) [0|7] "" NEO
 SG_ RLWheelDir : 37|3@0+ (1,0) [0|7] "" NEO
```

**Pull request merged**